pkg/extensions/sync/sync.go

```diff
@@ -338,7 +338,11 @@ func syncRegistry(regCfg RegistryConfig, storeController storage.StoreController
 
 	log.Info().Msg("filtering repos based on sync prefixes")
 
-	repos := filterRepos(catalog.Repositories, regCfg.Content)
+	repos, err := filterRepos(catalog.Repositories, regCfg.Content)
+	if err != nil {
+		log.Error().Err(err).Msg("error while filtering repos based on prefixes")
+		return err
```

Comment:
If we return this error here that will stop the sync on a Registry where the error occurs. Is this the desired behavior?
Maybe it is better to just log the error and continue. 
For me is something similar to stopping a server only because we have a bug somewhere in the client library.

Reply:
Hmm, I can do that, but currently the server won't start if there is any error in sync config. so this shouldn't ever happen.

Reply:
Ok, fixed that, thank you!